Engine/ResourceManagement/Importer/SkyboxImporter.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _SKYBOXIMPORTER_H_
+#define _SKYBOXIMPORTER_H_
+
+#include "ResourceManagement/Importer/Importer.h"
+#include "ResourceManagement/Resources/Skybox.h"
+
+#include <assimp/material.h>
```

Comment:
assimp for skybox?

Reply:
Yeah, that's wrong. Probably copy pasting code :D